3rdParty/iresearch/core/CMakeLists.txt

```diff
@@ -83,6 +83,14 @@ if (USE_OPTIMIZE_FOR_ARCHITECTURE)
   )
 endif()
 
+if (GENERATED_PARSER_SOURCES)
```

Comment:
the IResearch library must ensure that all its dependencies are built present, the first logic branch breaks this precondition

Reply:
The invoker overrules generation by specifying where to get the pre-generated file.

Reply:
this is not part of the scope of responsibility of the IResearch library, please provide a clean solution that does not cross scope of responsibility boundaries